gui/CameraView.cxx

```diff
@@ -912,17 +911,16 @@ void CameraView::enableAntiAliasing(bool enable)
 }
 
 //-----------------------------------------------------------------------------
-void CameraView::setRegistrationPointEditingEnabled(bool state)
+void CameraView::setEditMode(EditMode mode)
 {
   QTE_D();
 
-  d->UI.actionPlaceEditCRP->setEnabled(state);
-
-  if (!state)
-  {
-    d->UI.actionPlaceEditCRP->setChecked(false);
-    d->registrationPointsWidget->enableWidget(false);
-  }
+  d->groundControlPointsWidget->enableWidget(
```

Comment:
out of curiosity, what would happen if we enabled both widgets at the same time?